invenio_app_ils/ui/src/invenio_app_ils/common/urls.js

```diff
@@ -13,13 +13,18 @@ export const BackOfficeURLS = {
   patronDetails: `${BackOfficeBase}/users/:userPid`,
 };
 
-export const viewDetailsClickUrl = loanPid => {
+export const viewLoanDetailsClickUrl = loanPid => {
```

Comment:
Can you just omit the `Click` in the name?

Reply:
Yeah, I guess this function is more about creating/generating an URL Path given some params. It should be named accordingly.